### PR TITLE
Saved Note's Details sent from server

### PR DESCRIPTION
**Pull request by CAHILLSJ**
The changes in main.py loop over all the files in "/submitNote" and organize the title of the file, the contents, the date the file was created and last modified into a list for display on the website.

I've commented out lines 115-119 in main.js and line 28 on index.html
Remove the comments to make those lines active and a test button will appear on the website.
You can create and save notes in the submitNote file when you push the save button.
The new "Test" button will send all Notes saved from submitNote and can be found in the Network tab of you developer tools.

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -60,5 +60,30 @@ def submitNote():
...
+            data.update(Creation_Date=time.ctime(creatstamp))
+            data.update(Title=filetitle)
+            with open(filepath, 'r') as f:
+                mainbody = (f.read())
+            data.update(Contents=mainbody)
+            dataArray.append(data.copy())
+        jsonArray = json.dumps(dataArray)
+        return jsonArray
+    
+
```
Comment (sduman) [inline main.py]: The spacing here is a bit excessive, trim it down.

File: static/main.js [hunk 2]
```diff
@@ -30,47 +30,93 @@ function requestCallback(responseText)
...
+
+    })
+
+ });
+
+
 
 
 
+//-----FOR TESTING PURPOSES ONLY--------------------------
```
Comment (sduman) [inline static/main.js]: Put this in the PR itself, don't leave stuff like this in the source files.

File: templates/index.html [hunk 3]
```diff
@@ -23,7 +23,9 @@ <h1>QuickNotes</h1>
                             id="contents"></textarea>
       </div>
 
-      <button id="submit">Save</button> <button id="clear">Delete</button>
+      <button id="submit" onclick="javascript:buttonAction();">Save</button> <button id="clear">Delete</button>
+
+         <!-- <button type="button" id="test" onclick="javascript:test.onclick();">Test</button> -->
```
Comment (sduman) [inline templates/index.html]: Same as the JS stuff, don't leave this in here.

File: main.py (hunk 1 above)
Comment (CAHILLSJ) [reply main.py]: I deleted the excess white space.

File: static/main.js (hunk 2 above)
Comment (CAHILLSJ) [reply static/main.js]: I deleted the comments completely.  Testing works fine without a button needed

File: templates/index.html (hunk 3 above)
Comment (CAHILLSJ) [reply templates/index.html]: I deleted the comments completely. Testing works fine without a button needed

**Pull request merged**